raiden/utils/echo_node.py

```diff
@@ -0,0 +1,309 @@
+                # chose the winner
+                transfer = random.choice(tickets)
+                echo_amount = 49
+            else:
+                self.lottery_pool.put(transfer)
+
+        else:
+            log.debug(
+                'echo transfer received',
+                initiator=pex(transfer['initiator']),
+                amount=transfer['amount'],
+                identifier=transfer['identifier']
+            )
+            echo_amount = transfer['amount']
+
+        if echo_amount:
+            log.debug(
+                'sending echo transfer',
+                target=pex(transfer['initiator']),
+                amount=echo_amount,
+                orig_identifier=transfer['identifier'],
+                echo_identifier=transfer['identifier'] + echo_amount,
+                token_address=pex(self.token_address),
+            )
+
+            self.api.transfer_and_wait(
+                self.token_address,
+                echo_amount,
+                transfer['initiator'],
+                identifier=transfer['identifier'] + echo_amount
+            )
+
+    def stop(self):
+        self.stop_signal = True
+        gevent.wait(self.greenlets)
+
+
+@click.group(invoke_without_command=True)
+@options
+@click.option("--token-address", type=ADDRESS_TYPE, required=True)
+@click.pass_context
+def runner(ctx, **kwargs):
+    """ Start a raiden Echo Node that will send received transfers back to the initiator. """
+    # This is largely a copy&paste job from `raiden.ui.cli::run`, with the difference that
+    # an `EchoNode` is instantiated from the App's `RaidenAPI`.
+    slogging.configure(kwargs['logging'], log_file=kwargs['logfile'])
+
+    token_address = kwargs.pop('token_address')
+
+    (listen_host, listen_port) = split_endpoint(kwargs['listen_address'])
+    with socket_factory(listen_host, listen_port) as mapped_socket:
+        kwargs['mapped_socket'] = mapped_socket
+
+        app_ = ctx.invoke(app, **kwargs)
+
+        domain_list = []
+        if kwargs['rpccorsdomain']:
+            if ',' in kwargs['rpccorsdomain']:
+                for domain in kwargs['rpccorsdomain'].split(','):
+                    domain_list.append(str(domain))
+            else:
+                domain_list.append(str(kwargs['rpccorsdomain']))
+
+        raiden_api = RaidenAPI(app_.raiden)
+        if ctx.params['rpc']:
+            rest_api = RestAPI(raiden_api)
+            api_server = APIServer(
+                rest_api,
+                cors_domain_list=domain_list,
+                web_ui=ctx.params['web_ui'],
+            )
+            (api_host, api_port) = split_endpoint(kwargs["api_address"])
+            api_server.start(api_host, api_port)
+
+            print(
+                "The Raiden API RPC server is now running at http://{}:{}/.\n\n"
+                "See the Raiden documentation for all available endpoints at\n"
+                "https://github.com/raiden-network/raiden/blob/master"
```

Comment:
Change the URL like we also did in the normal app: `http://raiden-network.readthedocs.io/en/stable/rest_api.html`